### DISCUSSION TITLE
Alternative for Free Non-Premium accounts

### DISCUSSION THREAD

**Discussion by omnigenous** [hrkfdn/ncspot#1413] [Q&A]
Since ncspot is not planning on supporting this. Is there any alternatives?

**Comment by Bettehem** [comment]
It's not really up to for ncspot to decide. Spotify doesn't allow the use of many of the API endpoints for non-premium accounts, so a lot of ncspot's functionality would simply not be possible for free accounts. This is a limitation imposed by Spotify, not ncspot.
